### PR TITLE
feature(ui-ux): create wallet address input component

### PR DESCRIPTION
**Pull request by lykalabrada**
#### What this PR does / why we need it:
- Create wallet address input component 

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
Not included in this PR:
- ~~Address label with network tag (`MAINNET` or `TESTNET`)~~ UPD: Added
- ~~Icon tooltip (Paste from clipboard)~~ UPD: Added for tablet and web
- ~~Transparent background for the input~~ UPD: Done
- Update network tag from label to button - added on [PR#34](https://github.com/WavesHQ/bridge/pull/34) along with UI changes and integration to actual bridge form

#### Developer Checklist:

- [x] Read your code changes at least once
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode\*
- [x] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*
- [ ] Added translations\*

### CODE REVIEW

File: apps/web/src/components/WalletAddressInput.tsx [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    setError({ message, isError: hasInvalidInput });
+  }, [addressInput, isValidAddress, blockchain, network]);
+
+  useEffect(() => {
+    if (copiedFromClipboard) {
+      setTimeout(() => setCopiedFromClipboard(false), 2000);
+    }
+  }, [copiedFromClipboard]);
+
+  const showErrorBorder = addressInput && !isValidAddress && !isFocused;
```
Comment (kyleleow) [inline apps/web/src/components/WalletAddressInput.tsx:112]: we should show the error border even when the input is in focus

Comment (lykalabrada) [reply apps/web/src/components/WalletAddressInput.tsx]: Updated, thank you!

File: apps/web/src/components/WalletAddressInput.tsx [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+            }}
+          />
+        )}
+      </div>
+
+      {/* Error and warning messages */}
+      {error.message && !disabled && (
+        <span
+          className={clsx(
+            "px-4 pt-2 text-xs lg:px-6 lg:text-sm",
```
[the remark this answers is not in the document]
Comment (lykalabrada) [reply apps/web/src/components/WalletAddressInput.tsx]: Fixed 🙏

File: apps/web/src/components/WalletAddressInput.tsx [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+          containerClass={clsx("mr-3 lg:mr-6 shrink-0", {
+            "cursor-pointer hover:bg-dark-200 active:dark-btn-pressed":
+              !disabled,
+          })}
+          disableTooltip={disabled || !isSm} // Disable tooltip for mobile
+        >
+          <FiClipboard
+            size={20}
+            className="text-dark-1000"
+            onMouseDown={handlePasteBtnClick}
```
Comment (kyleleow) [inline apps/web/src/components/WalletAddressInput.tsx:159]: doesnt work on mobile

Comment (lykalabrada) [reply apps/web/src/components/WalletAddressInput.tsx:159]: Hmm working on iOS and Google Pixel for me, I think there's a permission setting to allow paste on mobile

Comment (kyleleow) [reply apps/web/src/components/WalletAddressInput.tsx:159]: will revisit next time

**Review by kyleleow — APPROVED** [review]

**Comment by lykalabrada** [comment]
Changes from this PR is added to a new PR https://github.com/WavesHQ/bridge/pull/34
Thanks for the review @kyleleow! 🙏

**Pull request closed without merging**